apollo-router/src/plugins/connectors/make_requests.rs

```diff
@@ -1709,6 +1709,13 @@ mod graphql_utils {
         variables: &Map<ByteString, JSONValue>,
     ) -> Result<Map<ByteString, JSONValue>, BoxError> {
         let mut arguments = Map::new();
+
+        for argument_def in field.definition.arguments.iter() {
+            if let Some(value) = argument_def.default_value.as_ref() {
+                arguments.insert(argument_def.name.as_str(), argument_value_to_json(value)?);
```

Comment:
Instead of always jsonifying up here, we could do `entry().or_insert_with()` after inserting the actual values. Also, this file length makes me itchy 😓

Reply:
move the module to its own file and took your suggestion. let me know if you have ideas on handling the fallible conversion fn!